src/schemas/21-526EZ/schema.js

```diff
@@ -15,139 +14,118 @@ const directDepositDef = {
       type: 'string',
       pattern: '^\\d{4,17}$'
     },
-    routingNumber: {
-      type: 'string',
-      pattern: '^\\d{9}$'
-    },
     bankName: {
       type: 'string',
-      maxLength: 35, // Is this going to be enough?
+      maxLength: 35,
       pattern: "([a-zA-Z0-9\-'.,# ])+$"
     }
   }
-}
+};
 
-// Copy pasta from the common address definition
-const countries = constants.countries.map(object => object.value);
-const countriesWithStateList = Object.keys(constants.states).filter(x => _.includes(countries, x));
-const countryStateProperties = _.map(constants.states, (value, key) => ({
-  properties: {
-    country: {
-      type: 'string',
-      'enum': [key]
-    },
-    state: {
-      type: 'string',
-      // TODO: state is only a two-character code
-      'enum': value.map(x => x.value)
-    }
-  }
-}));
-countryStateProperties.push({
-  properties: {
-    country: {
-      not: {
+const disabilitiesBaseDef = {
+  type: 'array',
+  maxItems: 100,
```

Comment:
Should we add `minItems: 1` here?

Reply:
It _should_ get a `minItems: 1` property through the merging that's happening in the `definitions` section. I wanted to leave it out of the base definition because this is one place where the `disabilities` and `secondaryDisabilities` schemas differ 😞

Reply:
Thanks for the insight.